src/main/java/coresearch/cvurl/io/request/CVurlProxyType.java

```diff
@@ -0,0 +1,20 @@
+package coresearch.cvurl.io.request;
+
+import java.net.Proxy;
+
+public enum CVurlProxyType {
+    HTTP(Proxy.Type.HTTP),
+    SOCKS(Proxy.Type.SOCKS),
+    DIRECT(Proxy.Type.DIRECT),
```

Comment:
what the use case of the DIRECT option?

Reply:
`DIRECT` should be used  to mark that request should not use proxy. For example user set client level proxy, but he wants this particular request to not use proxy. In such case user should use `CVurlProxy#noProxy` factory method, which creates `CVurlProxy` with `Proxy.Type.DIRECT` type.